src/unittest/unittestScripts.ts

```diff
@@ -12,6 +12,22 @@ import base64
 import json
 import traceback
 
+from pathlib import Path
+start_dir = sys.argv[2]
+if Path(start_dir + "/manage.py").is_file():
+    with open(start_dir + "/manage.py", "r") as management_file:
+        contents = management_file.readlines()
+        if any(True for line in contents if line.strip().replace('"""', '') == "Django\'s command-line utility for administrative tasks."):
```

Comment:
Is this only to check that this is a django app?

Reply:
the code at line 20? yes I want to make sure the file is generated by django-admin startproject